src/modules/interp_met/WindNinja.cpp

```diff
@@ -90,23 +92,75 @@ void WindNinja::init(mesh& domain)
     {
         if( itr.find("Ninja") != std::string::npos)
         {
-          ++N_windfield;
+            ++N_windfield;
         }
     }
     N_windfield /= 3; // _U, _V, and speedup maps
 
     LOG_DEBUG << "Found " << N_windfield << " windfields";
+    try
+    {
+        // see if we have a manually specified number
+        int user_nwindfield = cfg.get<int>("N_windfield");
```

Comment:
Here we are not accounting for a mistake that could be done by a user. Say we have 12 windfields in the mesh file (every 30 deg) and only one averaging radius. If the user asks for 8 wind field in the cfg file, the code will only consider the eight first wind direction and assume that they are distributed every 45 deg instead of 30 deg. No exception will be raised.

Reply:
I have remove cfg.N_windfields and correctly auto detect it, even in the presence of multiple _LAvg maps. Thus the user can never use fewer or greater # maps than what exists

Commit dfcb558f119522a7d76b622315e965c8b56f9ce3

Reply:
Thanks Chris. It looks good to me.